README.md

```diff
@@ -1,56 +1,40 @@
-# Getting Started with Create React App
+# SlimeVR UI
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+This is the GUI of SlimeVR, it uses the SolarXR protocol to communicate with the server and is completely isolated from the server logic.
 
-In the project directory, you can run:
+This project is written in Typescript and using Tauri as the backend. Tauri is written in Rust.
 
-### `npm start`
+# Prerequisites
```

Comment:
Only `SlimeVR UI` should have the primary heading, the rest should be two or more `#`